deploy-renku/entrypoint.sh

```diff
@@ -12,8 +12,14 @@ echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin
 # set up kube context and values file
 echo "$RENKUBOT_KUBECONFIG" > "$KUBECONFIG" && chmod 400 "$KUBECONFIG"
 
-# set up the values file
-printf "%s" "$RENKU_VALUES" | sed "s/<replace>/${RENKU_RELEASE}/" > $RENKU_VALUES_FILE
+export RENKU_VALUES_FILE="/renku-values.yaml"
+
+# merge the secret and clear value files
+printf "%s" "$RENKU_SECRET_VALUES" > $RENKU_VALUES_FILE
+yq eval-all --inplace '. as $item ireduce ({}; . * $item )' $RENKU_VALUES_FILE $RENKU_CLEAR_VALUES_FILE
```

Comment:
I think it is better to let helm take care of this if we simply pass the files on to the helm command in the python script that does `helm upgrade --install`. I.e. we can do `helm install -f non-secret.values.yaml -f secret-values.yaml --set extra-values-from-comment`. Then we dont have to do this `yq` magic stuff. Helm already handles this.